static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,341 @@
+        margin-bottom: 0;
+        width: auto;
+        float: none;
+    }
+
+    .block-share--gallery {
+        position: relative;
+
+        .block-share__item--facebook,
+        .block-share__item--twitter,
+        .block-share__item--pinterest {
+            background-color: rgba(0, 0, 0, 0);
+            border: 1px solid $neutral-2;
+
+            &:hover {
+                background-color: rgba(0, 0, 0, 0);
+            }
+
+            &.inline-icon, .inline-icon {
+                fill: $neutral-3;
+            }
+        }
+    }
+
+    .social-icon, .block-share__item {
+        background-color: rgba(0, 0, 0, 0);
+        border: 1px solid $neutral-2;
+
+        &:hover {
+            background-color: rgba(0, 0, 0, 0);
+        }
+
+        &.inline-icon, .inline-icon {
+            fill: $neutral-3
+        }
+    }
+
+    .gallery {
+        list-style: none;
+        margin: 0;
+    }
+
+    .gallery__item {
+        figure {
+            display: flex;
+            flex-direction: column-reverse;
+        }
+
+        // On mobile the images should be 100% of the width of the screen
+        @include mq($until: phablet) {
+            margin-left: -$gs-gutter;
+            margin-right: -$gs-gutter;
+        }
+
+        @include mq($until: mobileLandscape) {
+            margin-left: -$gs-gutter / 2;
+            margin-right: -$gs-gutter / 2;
+        }
+
+        @include mq(tablet) {
+            position: relative;
+            border-top: 0;
+            padding: 0;
+
+            figure {
+                flex-direction: row;
+            }
+        }
+    }
+
+    .gallery__figcaption {
+        @include fs-textSans(1);
+        padding-top: $gs-baseline / 4;
+        overflow: visible;
+        max-height: 100%;
+        position: relative;
+        border: 0;
+        width: inherit;
+
+        @include mq($until: desktop) {
+            margin-bottom: $gs-baseline * 2;
+        }
+
+        @include mq($until: phablet) {
+            padding: $gs-gutter;
+        }
+
+        @include mq($until: mobileLandscape) {
+            padding: $gs-gutter / 2;
+        }
+
+        @include mq(tablet) {
+            @include fs-textSans(2, true);
+            position: absolute;
+            width: gs-span(3);
+            top: 0;
+            right: auto;
+            margin-left: -(gs-span(3));
+        }
+    }
+
+    .gallery__caption {
+        color: $neutral-4;
+        margin-bottom: 0;
+
+        @include mq(tablet) {
+            padding-right: $gs-gutter / 2;
+        }
+
+        a {
+            color: $neutral-4;
+            border-bottom: 1px solid #767676;
+
+            &:hover {
+                border-bottom: 1px solid $neutral-7;
+                text-decoration: none;
+            }
+        }
+
+        > strong:first-child {
+            @include fs-headline(2);
+            color: $neutral-5;
+            display: block;
+            margin-bottom: $gs-baseline/2;
+
+            & + br {
```

Comment:
Once the `.new-gallery` class is removed it will be exactly like: https://github.com/guardian/frontend/blob/320b993b0bd6c51e97bd8ed9adc92cf52f571f41/static/src/stylesheets/module/content/_gallery.scss#L96